src/schema/article.js

```diff
@@ -0,0 +1,46 @@
+import { orderedList, bulletList, listItem } from 'prosemirror-schema-list';
+import { Schema } from 'prosemirror-model';
+import { schema } from 'prosemirror-markdown';
+
+export const fullSchema = new Schema({
+  nodes: {
+    doc: schema.spec.nodes.get('doc'),
+    paragraph: schema.spec.nodes.get('paragraph'),
+    blockquote: schema.spec.nodes.get('blockquote'),
+    horizontal_rule: schema.spec.nodes.get('horizontal_rule'),
+    heading: schema.spec.nodes.get('heading'),
+    code_block: schema.spec.nodes.get('code_block'),
+    text: schema.spec.nodes.get('text'),
+    image: schema.spec.nodes.get('image'),
```

Comment:
@fayazara I have added schema support here. Let me know if I need to comment this out.